schema/schema.go

```diff
@@ -1,13 +1,43 @@
 package schema
 
-import driver "github.com/sjindal-arista/Arista-QA/memory-driver"
+// Question -
+type Question struct {
+	q       string
+	quesID  string
+	user    string
+	tags    []string
+	answers []*Answer
+}
+
+// Answer -
+type Answer struct {
+	ansID    string
+	content  string
+	user     string
+	quesID   string
+	comments []*Comment
+	stats    *Stats
+}
+
+// Stats -
+type Stats struct {
+	views     int
+	upvotes   int
+	downvotes int
+}
+
+// Comment -
+type Comment struct {
+	comment string
+	user    string
+}
 
 // Driver defines basic func for all storage
 type Driver interface {
-	AddNewQuestion(newQ string, u *driver.User) error
```

Comment:
Replacing user object by UserId string makes sense but does it suffice as all information required?

Reply:
yes.. it should be..  if we want to give points to users, that can be derived from the schema we already have. Other than that all other user related fields are not relevant to our requirements.